packages/tests/src.ts/utils.ts

```diff
@@ -61,9 +61,103 @@ function equals(a: any, b: any): boolean {
     return a === b;
 }
 
+
+const getAccounts = () => {
+    return {
```

Comment:
The most usable accounts are [Alias ECDSA](
https://github.com/hashgraph/hedera-local-node/blob/main/src/hederaUtils.js#L5), let's add them here as well